kuma/api/v1/views.py

```diff
@@ -206,3 +207,19 @@ def whoami(request):
     }
 
     return JsonResponse(data)
+
+
+@never_cache
+@require_GET
+def search(request, locale):
+    qs = request.GET.get('q')
+    search = (WikiDocumentType.search()
```

Comment:
I don't know enough about Elasticsearch to review this, so I'm going to defer to @peterbe here.

Reply:
I still think we should do a followup bug to see if we can extract this functionality somewhere from `kuma/search/views.py`. I don't think we need to file that bug right now.